mjolnir/input/read_local_interaction.hpp

```diff
@@ -122,6 +123,136 @@ read_contact_interaction(const std::string& kind, const toml::value& local)
     }
 }
 
+// This is internal function for read_directional_contact_interaction function.
+template<typename traitsT, typename angle1_potentialT, typename angle2_potentialT>
+typename std::unique_ptr<LocalInteractionBase<traitsT>>
```

Comment:
`typename` is used to declare that a dependent qualified name (such as, `T::type`) is a type. Here, you don't need to add it here.
```suggestion
std::unique_ptr<LocalInteractionBase<traitsT>>
```